llvm/tools/llvm-readobj/ELFDumper.cpp

```diff
@@ -3908,7 +3933,8 @@ template <class ELFT> void GNUELFDumper<ELFT>::printRelocations() {
     HasRelocSections = true;
 
     std::string EntriesNum = "<?>";
-    if (Expected<size_t> NumOrErr = GetEntriesNum(Sec))
+    Expected<size_t> NumOrErr = GetEntriesNum(Sec);
+    if (NumOrErr)
```

Comment:
Why has this changed?

Reply:
Sorry. Unneeded change when I was figuring out the best way to add CREL support here.